circle.yml

```diff
@@ -15,6 +15,9 @@ dependencies:
     - echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
     - sudo apt-get update && sudo apt-get install yarn
     - echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+    # Make sure there is no cache
+    - npm cache clean
+    - yarn cache clean
```

Comment:
What caching advantages?? Few seconds? At best 1min, but if it fails we need to rebuild without cache so instead we need to wait approx. 15min